pkg/sql/sem/builtins/builtins.go

```diff
@@ -609,6 +610,41 @@ var builtins = map[string]builtinDefinition{
 		},
 	),
 
+	"gen_random_ulid": makeBuiltin(
+		tree.FunctionProperties{
+			Category: categoryIDGeneration,
+		},
+		tree.Overload{
+			Types:      tree.ArgTypes{},
+			ReturnType: tree.FixedReturnType(types.Uuid),
+			Fn: func(_ *tree.EvalContext, _ tree.Datums) (tree.Datum, error) {
+				t := time.Now().UnixNano()
```

Comment:
Should we use here `time.Now().UTC().UnixNano()` instead?

Reply:
Don't believe that's necessary.